src/nvim/tui/tui.c

```diff
@@ -1230,6 +1228,17 @@ static void show_termcap_event(void **argv)
   verbose_stop();  // flush now
 }
 
+/// Ask the terminal to send us the background color.
+static void ask_for_terminal_bg(UI *ui)
```

Comment:
Let's use consistent names where possible.

```suggestion
static void request_terminal_bg(UI *ui)
```